viewing_party/party.py

```diff
@@ -1,21 +1,125 @@
+# -----------------------------------------
 # ------------- WAVE 1 --------------------
+# -----------------------------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {}
+    if title and genre and rating:
+        movie_dict.update({"title": title})
+        movie_dict.update({"genre": genre})
+        movie_dict.update({"rating": rating})
+        return movie_dict
+    else:
+        return None   

```

Comment:
I'd suggest flipping the sense of the condition around to handle the special case (that something is falsy) quickly and get out, rather than having that dangling `else` at the end. This arrangement is referred to as a guard clause or a guard condition. 

Notice that it lets us move the main focus of the function (creating the new dict) out of an indented block, letting it stand out more clearly.

```python
def create_movie(movie_title, genre, rating):
    if not movie_title or not genre or not rating:
        return None

    movie_dict = {'title':movie_title, 'genre':genre, 'rating':rating}
    return movie_dict
```